content/spaces/helm-reference.md

```diff
@@ -7,137 +7,382 @@ description: Spaces Helm chart configuration values
 
 This reference provides detailed documentation on the Upbound Space Helm chart. This Helm chart contains configuration values for installation, configuration, and management of an Upbound Space deployment.
 
-## account
 
 {{< table "table table-striped" >}}
 
-| Key | Type | Default | Description |
-|-----|------|---------|-------------|
-| account | string | `"notdemo"` | The Upbound organization this installation is associated with. |
-
-{{</ table >}}
-
-## billing
 
-Configuration values for Spaces billing settings.
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
```

Comment:
We need to get these versions correct. 
Also, we may need to have some "versioned" view in the future, as not everyone is running the latest version.

Reply:
Not sure what the correct version here would be. This is from https://github.com/upbound/spaces/blob/c3d88b5e98cd656b215db9115b771d62574f59d8/cluster/charts/spaces/Chart.yaml#L4